automate/minimus_mdf_flow.py

```diff
@@ -335,6 +335,10 @@ def mint_doi_steps():
                         "publicationYear.$": "$.dataset_mdata.dc.publicationYear",
                         "url.$": "$.mdf_portal_link",
                     },
+                    "event": "publish",
```

Comment:
I think these need to go up in the `attributes` section, and not in the `Doi` parent?